src/state/digital-exchange/components/actions.js

```diff
@@ -20,6 +24,13 @@ export const setDEComponents = digitalExchangeComponents => ({
   },
 });
 
+export const setDEComponentsListViewMode = listViewMode => ({
```

Comment:
typo: should be `setDEComponentListViewMode` instead of `setDEComponentsListViewMode`